cli/flox/doc/flox.md

```diff
@@ -78,6 +78,14 @@ set -px XDG_DATA_DIRS "/nix/var/nix/profiles/default"
 ./include/general-options.md
 ```
 
+## flox Options
+
+`--version`
+:   Print `flox` version.
+
+`--prefix`
+:   Print `flox` installation prefix / Nix store path (flox internal use only).
```

Comment:
Question: do we want to document internal commands? Also, is it really a Nix store path or just the path of the `run` directory?

Reply:
Dropping